intuita-webview/src/fileExplorer/App.tsx

```diff
@@ -12,11 +12,10 @@ import SearchBar from '../shared/SearchBar';
 import ActionsHeader from './ActionsHeader';
 import { vscode } from '../shared/utilities/vscode';
 
-type MainViews = Extract<View, { viewId: 'treeView' }>;
+type MainViews = Extract<View, { viewId: 'fileExplorer' }>;
 
 function App() {
 	const [view, setView] = useState<MainViews | null>(null);
```

Comment:
`const [viewProps, setViewProps] = ...`

are you planning to use the `INITIAL_STATE` in this PR as well?